test/case/03-test-update-from-master

```diff
@@ -21,43 +21,43 @@ LOG=${0##*/}.log
 boilerplate_master=$(new_boilerplate_repo)
 
 override_boilerplate_version $boilerplate_master/boilerplate
-add_convention $repo test/test_base_convention
+add_convention $repo test/test-base-convention
 
-make update_boilerplate >$LOG 2>&1
+make update-boilerplate >$LOG 2>&1
 rc=$?
 cat $LOG
 if [ $rc -ne 0 ] ; then
 	exit $rc
 fi
-check_update $repo 03_check_after_init
+check_update $repo 03-check-after-init
```

Comment:
Should `check_update` be `check-update`?

Reply:
This is a bash function. I didn't rename any of those. (Is that even legal? /me tries... Yup. Still, not planning to rename all the bash functions everywhere, that okay?)